### PR TITLE
add possibility to choose visible columns in table from contentType

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: ApiBundle/Transformer/ContentTransformer.php [hunk 1]
```diff
@@ -52,6 +52,7 @@ public function transform($mixed)
 
         foreach ($mixed->getAttributes() as $attribute) {
             $facade->addAttribute($this->getTransformer('content_attribute')->transform($attribute));
+            $facade->addLinearizeAttribute($this->getTransformer('content_attribute')->transform($attribute));
```
Comment (nicolasThal) [inline ApiBundle/Transformer/ContentTransformer.php]: why do you do twice the same transformation?

File: Backoffice/LeftPanel/Strategies/ContentTypeForContentPanelStrategy.php [hunk 2]
```diff
@@ -30,7 +30,6 @@ public function __construct(ContentTypeRepositoryInterface $contentTypeRepositor
     public function show()
     {
         $contentTypes = $this->contentTypeRepository->findAllByDeletedInLastVersion();
-
```
Comment (nicolasThal) [inline Backoffice/LeftPanel/Strategies/ContentTypeForContentPanelStrategy.php]: let the empty line

**Comment by itkg-nanne** [comment]
correction done @nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**